ufw-docker

```diff
@@ -92,10 +107,11 @@ function ufw-docker--add-rule() {
     local INSTANCE_IP_ADDRESS="$2"
     local PORT="$3"
     local PROTO="$4"
+    local NETWORK="$5"
```

Comment:
Please change this line to `local NETWORK="${5:-}"`, otherwise the Docker image `ufw-docker-agent` will fail to run due to missing the last parameter. Thanks!

Reply:
And please add one more test case for the function `ufw-docker--add-rule`, only 4 parameters without the last one `NETWORK`, thank you.

Reply:
Done :)